charts/gardener-extension-registry-cache/templates/deployment.yaml

```diff
@@ -49,7 +49,6 @@ spec:
         - --disable-controllers={{ .Values.disableControllers | join "," }}
         - --ignore-operation-annotation={{ .Values.controllers.ignoreOperationAnnotation }}
         - --leader-election-id={{ include "leaderelectionid" . }}
-        - --leader-election-resource-lock={{ .Values.leaderElection.resourceLock }}
```

Comment:
`make generate` outputs several times:
```
WARNING: generate-internal-groups.sh is deprecated.
WARNING: Please use k8s.io/code-generator/kube_codegen.sh instead.
```

I guess in g/g we also didn't adapt to the change?